spec/controllers/new_admin/allocation_chart_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe NewAdmin::AllocationChartController do
+  let(:user) { create(:user) }
+
+  before do
+    allow(controller).to receive(:authenticate_user!)
+    allow(controller).to receive(:current_user).and_return(user)
+  end
+
+  describe 'GET #index' do
+    context 'when there are no allocations' do
+      it 'returns successful status' do
+        get :index
+
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'returns users within an empty allocation' do
+        get :index
+
+        expect(subject.instance_variable_get(:@allocations)).to include(having_attributes(user_id: user.id))
```

Comment:
What are you expecting here is reflecting what is described in the test? And I'm curious, why are you using the `instance_variable_get`?

Reply:
It is, the allocation object doesn't have an ID or any other fields besides the `user_id` so I called it an empty allocation.

I'm trying to get the `@allocations` variable, is there a better way of doing this?

Reply:
instead of verifying the instance variable, you could verify if the content of allocations is displayed in the rendered page